.github/workflows/go-lint.yml

```diff
@@ -0,0 +1,29 @@
+name: Go code linters
```

Comment:
Does it make sense to invoke this exclusively for changes in the Go source code directories + `go.work` etc?

Reply:
Good idea, fixed.